README.md

```diff
@@ -48,8 +48,11 @@ repository after the new release of your package is merged into the registry.
 
 Registered packages must have an [Open Source Initiative approved license](https://opensource.org/licenses),
 clearly marked via a `LICENSE.md`, `LICENSE`, `COPYING` or similarly named file in the package repository.
-Packages that wrap proprietary libraries are acceptable if the licenses of those libraries permit open
-source distribution of the Julia wrapper code.
+Packages that wrap proprietary libraries (or otherwise restrictive libraries) are
+acceptable if the licenses of those libraries permit open source distribution of the Julia wrapper code.
+The LICENCE file (and preferably the README) should mention the more restrictive license of the wrapped
```

Comment:
@SaschaMann 

```suggestion
The more restrictive license of the wrapped code:
1. MUST be mentioned in either the `THIRD_PARTY_NOTICE` file or the `LICENSE` file (preferably the `THIRD_PARTY_NOTICE` file).
3. SHOULD be mentioned in the `README` file.

Reply:
(Where's `2.`?) 

That addresses my concerns 👍

The file formats sometimes differ (no extension vs md etc). Given the `must`, that's something perhaps worth mentioning. But maybe that's too pedantic.